platform/osgi-web-util/src/main/java/org/motechproject/osgi/web/ModuleRegistrationData.java

```diff
@@ -396,6 +416,30 @@ public String getDocumentationUrl() {
         return DEFAULT_DOCS_URL.equals(documentationUrl) ? null : documentationUrl;
     }
 
+    /**
+     * Determines which tab of module should be default and loaded as first based on
+     * user permissions.
+     *
+     * @return the name of the default tab
+     */
+    @JsonIgnore
+    public String determineDefaultTab() {
+
+        if(StringUtils.isNotBlank(defaultURL)) {
```

Comment:
Is this correct? don't we want to override this in case the user doesn't have the permission?